user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Migrating_existing_systems/Migrating_the_general_database_to_a_DMS_Cassandra_cluster.md

```diff
@@ -55,6 +55,8 @@ During the migration, each DMA will go through the following stages:
 
 If your system does not use an indexing database yet or if it already uses a OpenSearch or Elasticsearch cluster connected to the DMS, you can run a regular migration as described below. However, if your system uses an Elasticsearch database installed on a DMA, follow the procedure in the next tab, "Running a migration with bespoke Elasticsearch data".
 
+1. Ensure all Failover pairs are switched to the main Agent (the first agent of the pair)
```

Comment:
The way we [currently explain Failover](https://docs.dataminer.services/user-guide/Advanced_Functionality/Failover/About_DMA_Failover.html), there is no such thing as a "main Agent", so it would be better to rephrase this. Is it the top Agent in the Failover configuration screen that they should switch to?